src/components/RightPane/Map/UCIMap.tsx

```diff
@@ -1,6 +1,7 @@
 import '../../../../node_modules/leaflet.locatecontrol/dist/L.Control.Locate.min.js';
 
 import Leaflet, { Control, LeafletMouseEvent } from 'leaflet';
+import 'leaflet.locatecontrol'
```

Comment:
What is this for?

Reply:
I get an error in dev because `Locate.Control isn't a constructor` importing the library extends the interface and resolves the error. I'm not sure why this isn't an issue in production, but it's better to be safe with this trivial fix